docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -524,34 +523,33 @@ In either case, you need to provide streams of tensor data and arrays of values
 
 This section explains the following three concepts: sample, batch and epoch.
 
-Let's assume a given model requires three inputs and two labels.
+Let's assume a given model requires three inputs and two labels. This has been reflected in the examples below:
 
 1. Sample
 
-A sample denotes a single pair of inputs/labels.
+    A sample denotes a single pair of inputs/labels.
```

Comment:
@gichan-jang, please check this suggestion. Code snippets should be preceded by an introductory line ending with a colon. 
```suggestion
    A sample denotes a single pair of inputs/labels. The example can be as follows: 
```

Reply:
Right. Updated.